### DISCUSSION TITLE
Video as source to Depth

### DISCUSSION THREAD

**Discussion by neuralnetjerk** [thygate/stable-diffusion-webui-depthmap-script#332] [General]
Do you have any documentation on how to video as a source (new) option works? I ran one short video, took 4 or 5 hours.  The end result was a depth map for each frame in the form of a video.  Would I use that to create stereo pairs of each frame and stitch it together to make a 3D video?

Would LOVE to drop in a short clip, have it run and the end result be a stereo 3d video.  But that wasn't what happened.

Thanks for all your hard work.

**Comment by sonakravtsenko3** [comment]
Thanks :)
You can use the the _Use custom/pregenerated DepthMap video_ option and then select _Generate stereoscopic image(s)_. It would be cool to have a wiki page for things like this in the future.
